Update code examples in asset guide - Closes #972 (#978)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/app-development/asset.adoc

  };
}

module.exports = { MyAsset };
----

> If the validation doesn't throw any errors, it means the validations has been successfull, and the data will be applied like defined in the `apply()` function.

Reviewer: ```suggestion
If the validation doesn't throw any errors, it means the validations have been successful, and the data will be applied as defined in the `apply()` function.
```
